csrankings.csv

```diff
@@ -10767,6 +10769,8 @@ Niranjan Balasubramanian,Stony Brook University,http://www3.cs.stonybrook.edu/~n
 Nirmala Shenoy,Rochester Institute of Technology,https://www.rit.edu/gccis/nirmala-shenoy,NOSCHOLARPAGE
 Nirupam Roy,University of Maryland - College Park,https://www.cs.umd.edu/~nirupam,kKyEuTcAAAAJ
 Nirupama Bulusu,Portland State University,http://web.cecs.pdx.edu/~nbulusu,xpyAqbUAAAAJ
+Nisar Ahmed,University of Colorado Boulder,https://cohrint.info/,HA3R65IAAAAJ
+Nisar R. Ahmed,University of Colorado Boulder,https://cohrint.info/,HA3R65IAAAAJ

```

Comment:
Same issue as above. https://www.colorado.edu/cs/faculty, no affiliation with CS mentioned on homepage.

Reply:
And https://experts.colorado.edu/display/fisid_153237 for Nisar.

Reply:
And to make the case even stronger, Nisar is currently the sole advisor of [this](https://bisraelsen.github.io/bio/) CS PhD student.